tools/getkeys/index.js

```diff
@@ -20,8 +78,7 @@ async function getKeys(keyVaultClient, rc, vaultName) {
                 p.push((async () => {
                     const response = await keyVaultClient.getSecret(vaultName, secretName);
                     const envName = secretName.split('-').join('__'); // secret name can't contain underscores
-                    console.log(`Setting environment variable ${envName}...`);
-                    await setEnv(envName, response.value);
+                    console.log(`  ${envName}`);
                     rc.secrets[envName] = response.value;
```

Comment:
Previously the code would append entries to the '.bashrc' or '.zshrc' file as individual keys arrived from the value.

The code now bulk appends all the keys when we're done downloading.

This was part of a refactor that allowed me to use the secrets cached in '~/.fluidtoolrc' while debugging rather than re-auth to Azure each time.